src-webview/data-viewer/StringView.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+import { WebviewCommand } from '../../src-shared/WebviewCommand';
+import { WebviewMessage } from '../../src-shared/WebviewMessage';
+import { KeyContentsField } from './KeyContentsField';
+import './styles.css';
+
+interface State {
+    key?: string;
+    value?: string;
+}
+
+export class StringView extends React.Component<{}, State> {
+    constructor(props: {}) {
+        super(props);
+        this.state = {};
+    }
+
+    componentDidMount(): void {
+        window.addEventListener('message', (event) => {
+            const message: WebviewMessage = event.data;
+            if (message.command === WebviewCommand.KeyName) {
+                const key = message.value as string;
+                this.setState({ key });
+            } else if (message.command === WebviewCommand.StringData) {
+                const value = message.value as string;
+                this.setState({ value });
+            }
+        });
+    }
+
+    render(): JSX.Element | null {
+        const { key, value } = this.state;
+
+        if (typeof key === 'undefined' || typeof value === 'undefined') {
+            return null;
+        }
+
+        return (
+            <div className="dataviewer-container">
+                <h2>{key} (string)</h2>
```

Comment:
any sense for if '(string)' should be localized?

Reply:
And I suppose the same question applies for the other data types too

Reply:
Yes, I created a localize function to do that. In the future we should look at using `vscode-nls` for localization.